applications/paikkatietoikkuna.fi/full-map/main.js

```diff
@@ -1,64 +1,69 @@
-import 'oskari-loader!../../../packages/paikkatietoikkuna/lang-overrides/bundle.js';
+import './index.js';
 
+import 'oskari-loader!oskari-frontend/packages/framework/bundle/coordinatetool/bundle.js';
```

Comment:
Check the order of the imports. Most of the bundles use classes from divmanazer but it's at the bottom. Also the mapfull/mapmodule are the first ones to be started but are not at the beginning here. Not sure if it's meaningful other than mapmodule needs to be before mapwfs2. Perhaps it would be clearer to import in mostly the same order as these are used?